templates/partner_campaign.html

```diff
@@ -0,0 +1,388 @@
+  }
+  #app-download-button {
+    text-indent: -9999px;
+    margin: 0 auto;
+  }
+  #rewards {
+    font-family: Lato, sans-serif;
+    font-size: 18px;
+    line-height: 1.44;
+    margin: 30px;
+  }
+  #rewards h2 {
+    font-family: Lato, sans-serif;
+    font-size: 18px;
+    font-weight: bold;
+  }
+  #screenshot1 {
+    text-align: center;
+  }
+  #screenshot1 img,
+  #screenshot2 img,
+  #screenshot2-desktop img {
+    /* these have some drop shadow on the right */
+    position: relative;
+    left: 15px;
+  }
+  #screenshot1 hr {
+    width: 100%;
+    margin: 0;
+  }
+  #screenshot2 {
+    text-align: center;
+  }
+  #screenshot2 img {}
+  #screenshot2 hr {
+    width: 100%;
+    margin: 0;
+  }
+  #legal {
+    margin: 54px 8%;
+  }
+  #legal p {
+    font-family: Lato, sans-serif;
+    font-size: 12px
+  }
+  #testing {}
+
+  .hidden-desktop {}
+  .hidden-mobile {
+    display: none;
+  }
+  .desktop-split {}
+  .float-desktop {}
+
+  @media only screen and (min-width: 600px) {
+    #screenshot1 hr,
+    #screenshot2 hr {
+      display: none;
+    }
+    #tagline p { font-size: 25px; }
+    #callout { font-size: 32px }
+    #callout .value { font-size: 88px }
+    #intro p { font-size: 18px; }
+    #partner-campaign-header .partner.logo { text-align: left; }
+    #partner-campaign-header .origin.logo { text-align: right; }
+
+    #screenshot1 img,
+    #screenshot2 img,
+    #screenshot2-desktop img { left: 75px; }
+
+    .qrcode-wrapper p {
+      font-family: Lato, sans-serif;
+      font-size: 14px;
+      line-height: 26px;
+      text-align: center;
+    }
+    .qrcode-wrapper img {
+      margin: 0 auto;
+      margin-top: 10px;
+    }
+    .qrcode-wrapper {
+      background: #fff;
+      color: #000;
+      padding: 5px;
+      margin: 0 auto;
+      width: 190px;
+      height: 190px;
+      border-radius: 20px;
+    }
+    .hidden-mobile {
+      display: inherit;
+    }
+    .hidden-desktop {
+      display: none;
+    }
+    .desktop-split {
+      display: block;
+      max-width: 1024px;
+      margin: 0 auto;
+    }
+    .float-desktop {
+      float: left;
+      width: 50%;
+      margin: 0;
+    }
+    .page-splitter {
+      width: 100%;
+      margin: 50px auto;
+    }
+  }
+</style>
+{% endblock %}
+
+{% block extra_scripts %}
+  <script src="/static/js/qrcode.min.js" type="text/javascript"></script>
+  <script>
+    const isIOS = navigator.platform && /iPad|iPhone|iPod/.test(navigator.platform)
+    const referalCode = 'op:{{ referal_code }}'
+    const me = `https://www.originprotocol.com/partner/{{ referal_code }}`
+
+    const qrOptions = {
+      text: me,
+      width: 140,
+      height: 140,
+      colorDark : "#000000",
+      colorLight : "#ffffff",
+      correctLevel : QRCode.CorrectLevel.H
+    }
+    const qrcode1 = new QRCode('qrcode1', qrOptions)
+    const qrcode2 = new QRCode('qrcode2', qrOptions)
+
+    /**
+     * Copy a string to the clipboard
+     */
+    function codeToClipboard(code) {
```

Comment:
You've already done this part obviously. For future, you could use a npm package for this. 
https://www.npmjs.com/package/copy-to-clipboard

Can be installed as npm or manually copied into our repo. From the first glance they seem to cover some edge cases and have fallback functionality.

Reply:
No bundles in this project, it's not that complex, and it was used as inspiration.  The only missing edge case is IE, which isn't a player in our case.

Reply:
👍